packages/components/src/tabs/src/Tabs.css

```diff
@@ -166,7 +199,7 @@
 
 /* TAB | STATE | DISABLED */
 .o-ui-tab[disabled] {
-    opacity: var(--o-ui-disabled-opacity);
+    color: var(--hop-neutral-text-disabled);
```

Comment:
I see you added the disabled color here, but it is difficult to notice. I'm thinking maybe we should have the disabled abckground color as well?
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/3e46a775-14a2-470f-96d1-11924008a573)

The disabled state doesn't seem to be in figma, only for the other style of tabs.

Reply:
I agree that it's not optimal. As Hopper has no disabled states, there's not a lot of risk here that it'll ever see the light of the day. Adding a token because of Orbit seems counter productive.